contracts/modules/SocialRecoveryModule.sol

```diff
@@ -1,18 +1,18 @@
-pragma solidity 0.4.19;
-import "../Extension.sol";
-import "../GnosisSafe.sol";
+pragma solidity 0.4.23;
+import "../Enum.sol";
+import "../Module.sol";
+import "../ModuleManager.sol";
+import "../OwnerManager.sol";
 
 
-/// @title Social Recovery Extension - Allows to replace an owner without Safe confirmations if friends approve the replacement.
+/// @title Social Recovery Module - Allows to replace an owner without Safe confirmations if friends approve the replacement.
 /// @author Stefan George - <stefan@gnosis.pm>
-contract SocialRecoveryExtension is Extension {
+contract SocialRecoveryModule is Module {
 
-    string public constant NAME = "Social Recovery Extension";
+    string public constant NAME = "Social Recovery Module";
     string public constant VERSION = "0.0.1";
-    bytes4 public constant REPLACE_OWNER_FUNCTION_IDENTIFIER = hex"54e99c6e";
+    bytes4 public constant REPLACE_OWNER_FUNCTION_IDENTIFIER = hex"e318b52b";
```

Comment:
Consider calc. identifier inline for transparency: bytes4 public constant CALL_PREFIX = bytes4(keccak256("callGasRelay(address,uint256,bytes32,uint256,uint256,address)"));